README.rst

```diff
@@ -31,9 +31,8 @@ Apache Libcloud - a unified interface into the cloud
 .. image:: https://coveralls.io/repos/github/apache/libcloud/badge.svg?branch=trunk
         :target: https://coveralls.io/github/apache/libcloud?branch=trunk
 
-Apache Libcloud is a Python library which hides differences between different
-cloud provider APIs and allows you to manage different cloud resources
-through a unified and easy to use API.
+Apache Libcloud is a Python library that wraps many different
+cloud provider APIs including: ...
```

Comment:
There's no follow on to this sentence, it doesn't make any sense now. I'm closing this PR.